crates/ironclaw_gateway/static/theme.css

```
@@ -1,34 +1,199 @@
+  --accent-soft: var(--v2-accent-soft);
+  --success: var(--v2-positive-text);
+  --info: var(--v2-info-text);
+  --warning: var(--v2-warning-text);
+  --danger: var(--v2-danger-text);
+  --code-bg: var(--v2-code-bg);
+  --radius: var(--v2-radius-sm);
+  --radius-lg: var(--v2-radius-lg);
   --shadow: 0 2px 8px rgba(0, 0, 0, 0.4);
-  --font-mono: 'IBM Plex Mono', 'SF Mono', 'Fira Code', Consolas, monospace;
-  --bg-overlay: rgba(0, 0, 0, 0.5);
+  --font-sans: 'Geist', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)